### PR TITLE
added operation path options

### LINKED ISSUE #18: Doc -  referring to full op\flow path without import
Add explanation about new feature - referring to full op\flow path without import

### PR DESCRIPTION
**Pull request by SamMarkowitz**
fixes #18

### CODE REVIEW

File: docs/cloudslang_dsl_reference.md [hunk 1]
```diff
@@ -349,7 +349,14 @@ inputs:
 The key `do` is a property of a [task](#task) name, a [loop](#loop), or an [async_loop](#async_loop).
 It is mapped to a property that references an [operation](#operation) or [flow](#flow).
 
-Calls an [operation](#operation) or [flow](#flow) and passes in relevant [input](#inputs). The [operation](#operation) or [flow](#flow) may be called by its qualified name using an alias created in the [imports](#imports) parameter.
+Calls an [operation](#operation) or [flow](#flow) and passes in relevant [input](#inputs). 
+
+The [operation](#operation) or [flow](#flow) may be called in several ways:
+
++ by referencing the [operation](#operation) or [flow](#flow) by name when it is in the default namespace (the same namespace as the calling [flow](#flow))
++ by using a fully qualified name (e.g. `path.to.operation.op_name`)
```
Comment (levice14) [inline docs/cloudslang_dsl_reference.md:357]: maybe specify here that a path in only recognized as full path if the the prefix `path` (first word before `.` character) is not a defined alias (cannot be expanded)
WDYT?

File: docs/cloudslang_dsl_reference.md [hunk 2]
```diff
@@ -349,7 +349,14 @@ inputs:
 The key `do` is a property of a [task](#task) name, a [loop](#loop), or an [async_loop](#async_loop).
 It is mapped to a property that references an [operation](#operation) or [flow](#flow).
 
-Calls an [operation](#operation) or [flow](#flow) and passes in relevant [input](#inputs). The [operation](#operation) or [flow](#flow) may be called by its qualified name using an alias created in the [imports](#imports) parameter.
+Calls an [operation](#operation) or [flow](#flow) and passes in relevant [input](#inputs). 
+
+The [operation](#operation) or [flow](#flow) may be called in several ways:
+
++ by referencing the [operation](#operation) or [flow](#flow) by name when it is in the default namespace (the same namespace as the calling [flow](#flow))
```
Comment (levice14) [inline docs/cloudslang_dsl_reference.md:356]: add example here? `e.g. op_with_default_namespace`

or maybe put a standalone section / flow with examples for the 4 cases (do we have other examples somewhere?)

**Comment by levice14** [comment]
:+1: 

gr8 description.. I am just concerned that is not enough info /  examples here for a new CloudSlang user

**Comment by levice14** [comment]
example section looks great :wink: 

maybe add a link to the example section in the DSL reference (not sure about this)? WDYT?

**Comment by SamMarkowitz** [comment]
Ah yes, meant to do that. Forgot.

**Comment by levice14** [comment]
@SamMarkowitz did you see my upper comments? (asking since you assigned the issue to me)

**Comment by SamMarkowitz** [comment]
Yes. I think I took care of them. Added a note about the prefix not being an alias and added an example to the examples page.

**Comment by levice14** [comment]
I mean the following inline comments:
- https://github.com/CloudSlang/docs/commit/8ff36ccae67fcec5d7c9740b7920ab374a618f14#commitcomment-13842958
- https://github.com/CloudSlang/docs/commit/8ff36ccae67fcec5d7c9740b7920ab374a618f14#commitcomment-13843610
- https://github.com/CloudSlang/docs/commit/8ff36ccae67fcec5d7c9740b7920ab374a618f14#commitcomment-13843563
- https://github.com/CloudSlang/docs/commit/8ff36ccae67fcec5d7c9740b7920ab374a618f14#commitcomment-13843619

**Comment by SamMarkowitz** [comment]
Made the fixes. Good eye.

The links are correct.

**Comment by levice14** [comment]
:+1:

**Pull request merged**